module/zfs/vdev_label.c

```diff
@@ -1876,7 +1877,10 @@ vdev_extra_sync(zio_t *zio, uint64_t *good_writes, vdev_t *vd, int flags,
 	if (!vdev_writeable(vd))
 		return;
 
-	// TODO Invoke extra sync logic for anyraid
+	if (vd->vdev_parent->vdev_ops == &vdev_anyraid_ops) {
+		vdev_anyraid_write_map_sync(vd, zio, txg, good_writes, flags,
+		    status);
```

Comment:
I suppose this will write all the maps every time?  In addition to 2 one-sector uberblock writes per leaf we now write up to 64MB per vdev?